internal/pkg/cli/deploy/lbws.go

```diff
@@ -216,6 +235,33 @@ func (d *lbWebSvcDeployer) validateALBRuntime() error {
 	return nil
 }
 
+func (d *lbWebSvcDeployer) validateImportedALBConfig() error {
+	if d.lbMft.HTTPOrBool.ImportedALB == nil {
+		return nil
+	}
+	alb, err := d.elbGetter.LoadBalancer(aws.StringValue(d.lbMft.HTTPOrBool.ImportedALB))
+	if err != nil {
+		return fmt.Errorf(`retrieve load balancer %q: %w`, aws.StringValue(d.lbMft.HTTPOrBool.ImportedALB), err)
+	}
+	if len(alb.Listeners) == 0 || len(alb.Listeners) > 2 {
+		return fmt.Errorf(`imported ALB %q must either one or two listeners`, aws.StringValue(d.lbMft.HTTPOrBool.ImportedALB))
```

Comment:
nit: I think including the ARN might be slightly more helpful in determining the problem alb!
```suggestion
		return fmt.Errorf(`imported ALB %q must have either one or two listeners`, alb.ARN))
```